config/settings.yml

```diff
@@ -18,3 +18,18 @@ mainnet_cota_regular_code_hash: "0x1122a4fb54697cf2e6e3a96c9d80fd398a936559b9095
 
 nrc_721_factory_output_data_header: "0x24ff5a9ab8c38d195ce2b4ea75ca8987"
 nrc_721_token_output_data_header: "0x0ddeff3e8ee03cbf6a2c6920d05c381e"
+
+default_hash_length: "64"
```

Comment:
These settings are specific to mainnet. In testnet, these settings are different.
We should split into mainnet config file and testnet config file